Network/load-balancing-comparison-matrix.md

```diff
@@ -49,5 +49,5 @@ The links below provide supporting material that can assist a customer with furt
 * [Citrix Netscaler VPX Documentation](http://docs.citrix.com/en-us/netscaler/11-1.html)
 * [Citrix Netscaler VPX Editions](//www.citrix.com/products/netscaler-adc/platforms.html#editions)
 * [CenturyLink Ecosystem Partners](../Ecosystem Partners/General/ecosystem-partner-list.md)
-* [CenturyLink Pricing Catalog](//ctl.io/pricing)
+* [CenturyLink Pricing Catalog](//www.ctl.io/pricing/#/va1)
```

Comment:
we should not have VA1 on this url.  its https://www.ctl.io/pricing as it auto finds where you are in the world